tests/tracer/test_propagation.py

```diff
@@ -1251,8 +1240,37 @@ def test_extract_tracecontext(headers, expected_context):
 ]
 
 
-@pytest.mark.parametrize("name,styles,headers,expected_context", EXTRACT_FIXTURES)
-def test_propagation_extract(name, styles, headers, expected_context, run_python_code_in_subprocess):
+EXTRACT_FIXTURES_ENV_ONLY = [
+    (
+        # b3 will only override to b3multi when set via envar
+        "test_deprecated_b3_style_still_works",
+        ["b3"],
+        B3_HEADERS_VALID,
+        {
+            "trace_id": 5208512171318403364,
+            "span_id": 11744061942159299346,
+            "sampling_priority": 1,
+            "dd_origin": None,
+        },
+    ),
+    (
+        # tracecontext propagation sets additional meta data that
+        # can't be tested correctly via test_propagation_extract_w_config. It is tested separately
```

Comment:
why not?

Reply:
maybe a comment above before `EXTRACT_FIXTURE_ENV_ONLY =` would be helpful for why we are singling out this set of options.

how would someone know where to add a fixture in the future?

Reply:
"why not?" Because you can't input values for tracecontext propagation and only get 
```
      "trace_id": context.trace_id,
      "span_id": context.span_id,
      "sampling_priority": context.sampling_priority,
      "dd_origin": context.dd_origin,
```
as expected context, while also getting the same output as doing  `assert context == Context(**expected_context)`. IIRC the issue was that we always store additional metadata such as tracestate and traceparent.

Reply:
I'll add a comment